docs/administration/configuration/configuring-interactive-sessions/index.md

```diff
@@ -0,0 +1,53 @@
+# Configuring interactive sessions
+
+## Interactive sessions cleanup
+
+### Automatic and scheduled closure
+
+By default, interactive sessions in REANA are kept open indefinitely. However, administrators
+have the flexibility to configure the maximum inactivity period after which interactive
+sessions will be automatically closed; the schedule of this cleanup process can be
+customized. This helps optimize resource utilization and ensure that unused sessions
```

Comment:
```suggestion
can configure a maximum inactivity period after which interactive
sessions will be automatically closed. This helps optimize resource utilization and ensure that unused sessions
```

WYDT? I don't think it's necessary to already talk about schedule here, it  can be introduced later when we show the Helm values.